manifests/0000_70_dns-operator_00-custom-resource-definition.yaml

```diff
@@ -1,20 +1,22 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: dnses.operator.openshift.io
+  creationTimestamp: null
+  name: dns.operator.openshift.io
 spec:
   group: operator.openshift.io
   names:
     kind: DNS
-    listKind: DNSList
-    plural: dnses
-    singular: dns
-  scope: Cluster
-  version: v1
-  subresources:
-    status: {}
```

Comment:
Won't this break status reporting?

Reply:
Will be fixed in https://github.com/openshift/api/pull/336